main_routers/live2d_router.py

```
@@ -0,0 +1,911 @@
+        live2d_model_name = None
+        model_info = None
+        
+        # 尝试通过item_id查找
+        if item_id:
+            try:
+                all_models = find_models()
+                matching_model = next((m for m in all_models if m.get('item_id') == item_id), None)
+                if matching_model:
+                    model_info = matching_model.copy()
+                    live2d_model_name = model_info['name']
+            except Exception as e:
```

Ruff (0.14.8)

Do not catch blind exception: `Exception`

(BLE001)